validators/cgroup_validator_linux.go

```diff
@@ -1,3 +1,4 @@
+//go:build linux
```

Comment:
with this change are we bumping the minimum required go version?
https://github.com/kubernetes/system-validators/blob/master/go.mod#L3

Reply:
bump the go version to 1.16?

Reply:
Can we avoid changing these go directives? Did you get an error without the change?

Reply:
@pacoxu this question? ^